Sources/SwiftSyntax/generated/syntaxNodes/SyntaxDeclNodes.swift

```diff
@@ -3772,7 +3784,7 @@ public struct MacroExpansionDeclSyntax: DeclSyntaxProtocol, SyntaxHashable {
     self.init(data)
   }
   
-  public var unexpectedBeforePoundToken: UnexpectedNodesSyntax? {
+  public var unexpectedBeforeAttributes: UnexpectedNodesSyntax? {
```

Comment:
Note to self: If we want to maintain source stability, we would need to add a deprecated compatibility accessor for this property. I think it’s fine for this PR though.